data/brands/shop/toys.json

```diff
@@ -344,6 +344,17 @@
         "name": "myToys",
         "shop": "toys"
       }
+    },
+        {
+      "displayName": "Noriel",
+      "id": ,
```

Comment:
This is broken json, but for future reference `id` is auto generated, you can ignore it when contributing.

```suggestion
```